index.js

```diff
@@ -12,7 +12,7 @@ const PLUGIN_NAME = 'gulp-sass';
 //////////////////////////////
 // Main Gulp Sass function
 //////////////////////////////
-const gulpSass = (options, sync) => through.obj((file, enc, cb) => { // eslint-disable-line consistent-return
+const gulpSass = (options, sync) => transfob((file, enc, cb) => { // eslint-disable-line consistent-return
```

Comment:
🔍 [`transfob`](https://github.com/metafizzy/transfob) is a lighter-weight alternative to `through2`.

Reply:
Cool